assets/src/edit-story/components/keyboardShortcutsMenu/constants.js

```diff
@@ -32,8 +37,8 @@ export const SPECIAL_KEYS = {
   ENTER: { symbol: '⏎', title: __('Enter', 'web-stories') },
   SHIFT: { symbol: '⇧', title: __('Shift', 'web-stories') },
   OPTION: { symbol: '⌥', title: __('Option', 'web-stories') },
-  ALT: 'Alt',
+  ALT: __('Alt', 'web-stories'),
```

Comment:
@swissspidy, do you want a translator comment or context on this one? And maybe the other keys as well? They all seem fairly ambiguous to me.

Reply:
context everywhere sounds reasonable 👍

Reply:
@swissspidy, what context message would be appropriate? I'm not familiar with the best-practice lingo here. `'The keyboard key "Enter"'` or?

Reply:
@barklund that works!

Reply:
Done in 62e81c2c4.